distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -577,6 +585,25 @@ public void testCswXmlIngest() {
     }
 
     @Test
+    public void testCswXmlWithHeaderIngest() {
```

Comment:
I don't think that is being done anywhere, but this itest is more-so to illustrate that in future itests, test data doesn't need to be stripped of its XML headers anymore for it to be used (a problem I've run into on a few occasions)